<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbSettingsDialog</class>
 <widget class="QDialog" name="DbSettingsDialog">
  <property name="windowTitle">
   <string>Подключение к базе данных</string>
  </property>
  <property name="minimumWidth">
   <number>420</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Хост:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="hostEdit">
       <property name="placeholderText">
        <string>localhost или IP-адрес</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Порт:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="portEdit">
       <property name="maximumWidth">
        <number>80</number>
       </property>
       <property name="placeholderText">
        <string>5432</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="databaseLabel">
       <property name="text">
        <string>База данных:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="databaseEdit">
       <property name="placeholderText">
        <string>имя базы данных</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Пользователь:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="usernameEdit">
       <property name="placeholderText">
        <string>postgres</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>пароль</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="testButton">
     <property name="text">
      <string>Проверить подключение</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
